styleguide/components/21-atoms/list/_list.scss

```diff
@@ -13,6 +13,19 @@ ol {
     @extend %list-no-style;
   }
 
+  &.inline {
+    @extend %list-no-style;
+
+    display: flex;
```

Comment:
Is het verstandig om deze op full width default te doen?

Reply:
Don't know... waarom niet?